src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -141,7 +151,8 @@ private boolean setupDependencies() throws IOException {
         Path defaultJavaStaticPath = configuration.getDefaultJavaStaticPath();
         boolean customJavaLocation = configuration.useCustomJavaStaticLibs();
 
-        boolean downloadJavaStatic = false, downloadJavaFXStatic = false;
+        boolean downloadJavaStatic = false, downloadJavaFXStatic = false, downloadAndroidSdk = false,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4992870)